src/Enzo/enzo_EnzoMethodPpml.cpp

```diff
@@ -28,6 +28,14 @@ EnzoMethodPpml::EnzoMethodPpml()
   const EnzoDualEnergyConfig& de_config = fluid_props->dual_energy_config();
   ASSERT("EnzoMethodPpml::EnzoMethodPpml",
          "incompatible with dual energy formalism", de_config.is_disabled());
+
+  // technically, the PPM solver doesn't directly use any of the functionality
```

Comment:
It should probably say PPML instead of PPM in this entire code block, right?

Reply:
You're absolutely right. I'll fix that